src/annotation.js

```diff
@@ -165,13 +164,17 @@ function drawElements(ctx, elements, caller) {
     if (el.options.drawTime === caller) {
       el.draw(ctx);
     }
+    drawSubElements(ctx, el, caller);
   }
 }
 
-function drawSubElements(ctx, elements, caller) {
-  for (const el of elements) {
-    if (isArray(el.elements)) {
-      drawElements(ctx, el.elements, caller);
+function drawSubElements(ctx, el, caller) {
+  if (el.elements && el.elements.length) {
+    const box = 'getBoundingBox' in el ? el.getBoundingBox() : undefined;
```

Comment:
@kurkle maybe you don't like it. I prefer to pass an undefined value instead of a default. Let me know.

Reply:
You are right, I don't like it. If we always passed an area (be it canvas area or chart area), it removes one `if` from all of the elements that need to do clipping (or whatever) based on the provided area.